### PR TITLE
4317 scenario 2.1 basic service

### PR DESCRIPTION
**Pull request by LauraStordeur**
# Description

Added the first part of the Service Catalog scenario. I used a few methods to await on the compile to be finished. I suppose we can optimise them maybe, but we are tied to the asyncness of Cypress. While loops are not supported. 

closes *#4317*

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Code is clear and sufficiently documented
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

File: cypress/e2e/scenario-2.1-basic-service.cy.js [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    cy.visit("/console/");
+
+    // Add interceptios for the delete and get call to be able to catch responses later on.
+    cy.intercept("DELETE", "/lsm/v1/service_inventory/basic-service/**").as(
+      "DeleteInstance"
+    );
+    cy.intercept(
+      "GET",
+      "/lsm/v1/service_inventory/basic-service?include_deployment_progress=True&limit=20&&sort=created_at.desc"
+    ).as("GetServiceInventory");
```
Comment (matborowczyk) [inline cypress/e2e/scenario-2.1-basic-service.cy.js:217]: If I am not mistaken, as you don't use it @GetServiceInventory further in that test case, there is no need to "intercept.as" just intercept

**Review by matborowczyk — APPROVED** [review]
beside that one minor thing, which I am only verifying all good :)

File: cypress/e2e/scenario-2.1-basic-service.cy.js (hunk 1 above)
Comment (LauraStordeur) [reply cypress/e2e/scenario-2.1-basic-service.cy.js]: You're right, I had forgotten to remove it :D

Comment (LauraStordeur) [reply cypress/e2e/scenario-2.1-basic-service.cy.js:229]: this is now fixed :)

**Review by bartv — APPROVED** [review]

**Pull request closed without merging**